packages/portalnetwork/src/networks/history/history.ts

```diff
@@ -131,7 +148,15 @@ export class HistoryNetwork extends BaseNetwork {
       }
       if (Array.isArray(proof.value)) {
         try {
-          verifyPreMergeHeaderProof(proof.value, contentHash, header.number)
+          if ('blockHash' in validation) {
+            verifyPreMergeHeaderProof(proof.value, validation.blockHash, header.number)
+          } else {
+            verifyPreMergeHeaderProof(
```

Comment:
Here's a theoretical question for us down the road.  Why are we proving things against a reconstructed `epochAccumulator` if we don't even have access to them any more.  Not to be solved now but I'm just curious.